leads/views.py

```diff
@@ -240,6 +240,24 @@ def lead_history(request, lead_id):
     history_entries = LeadHistory.objects.filter(lead=lead, category='mention')
     return render(request, 'lead/lead_history.html', {'lead': lead, 'history_entries': history_entries})
 
+def send_appointment_reminder(lead):
+    
+    reminder_time = lead.appointment_date_time - timedelta(minutes=15)
+    current_time = datetime.now()  # Use the current time
+
+    print("Current Time:", current_time)
+    print("Reminder Time:", reminder_time)
```

Comment:
Remove these print

Reply:
its for you to check it if its working or not

Reply:
so you didnnt test it?